gradle/libs.versions.toml

```diff
@@ -23,7 +23,7 @@ gradle-goomph-plugin = "com.diffplug.gradle:goomph:3.44.0"
 gradle-maven-publish-plugin = "com.vanniktech:gradle-maven-publish-plugin:0.29.0"
 gradle-spotless-plugin = { module = "com.diffplug.spotless:spotless-plugin-gradle", version.ref = "spotless" }
 gson = "com.google.code.gson:gson:2.11.0"
-guava = "com.google.guava:guava:33.4.5-jre"
+guava = "com.google.guava:guava:33.4.7-jre"
```

Comment:
Opportunistically bump Guava; the 33.4.5-jre version had a couple of minor issues that have been fixed